src/scss/design-system-interface.scss

```diff
@@ -49,18 +49,19 @@ $c-pale: #F5F1F6;
   &__title {
     margin: 0;
 
-    font-family: 'Roboto', sans-serif;
-    font-size: 14px;
-    font-weight: bold;
+    font-family: $primary-font;
+    font-size: 1rem;
+    font-weight: 900;
```

Comment:
I'm wondering if we should have some sort of `font-weight` utility?

Reply:
What do you mean by utility @colbyallenc? As in a utility class or mixin or function to output the value?

Reply:
I wasn't sure if there was a way to come up with a system similar to the way we do `font-size` where all of the numbers we want to use is set as a variable to be reused instead of using a number value.